package.json

```diff
@@ -38,14 +38,13 @@
     "homepage": "https://github.com/aws/aws-vsts-tools",
     "devDependencies": {
         "@types/adm-zip": "^0.4.33",
-        "@types/archiver": "^3.0.0",
         "@types/base-64": "^0.1.3",
         "@types/fs-extra": "^8.0.0",
         "@types/jest": "^26.0.19",
         "@types/js-yaml": "^3.12.1",
         "@types/json-query": "^2.2.0",
         "@types/node": "^10.17.28",
-        "@types/q": "^1.5.2",
+        "@types/q": "^1.5.4",
```

Comment:
do we still need this if `q` is dropped from dependencies?

Reply:
Yeah I wish I could put comments in here. `azure-pipelines-task-lib` requires this to build....but doesn't have it in its own dependencies (nice)